src/components/SortDropDown.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import Button from '@mui/material/Button';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Fade from '@mui/material/Fade';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import { useContext } from "react"
+import { PetsContext, SearchContext } from "../context"
+import sortPetList from './sortPetList';
+
+const SortDropDown = () => {
```

Comment:
The logic in this component is good and works well. I do see some opportunities to remove some duplicate code. Instead of explicitly defining menu items and methods for each type, you could try something like this:
```
const sortOptions = [
   {
      key: 'distance',
      label: 'Distance'
   }
   /// the rest
]

const handleSortChange = (value) => {
   sortPetList(type, zipcode, `sort=${value}`).then do the rest of the stuff
}

{ sortOptions.map(so => <MenuItem onClick={() => handleSortChange(so.key)}>{ so.label }</MenuItem>)}
```
Obviously there are some blanks to be filled in but this would allow the `sortOptions` list to determine each of the MenuItems driven by a single `handleSortChange` method, which will make it very easy to add additional sort options in the future.